package.json

```
@@ -69,7 +69,9 @@
     "test:unit:watch": "yarn test:unit -- --watch",
     "test:unit:updatesnapshots": "NODE_OPTIONS=--experimental-vm-modules jest --updateSnapshot --env=jsdom --coverage --silent=false --colors --testPathIgnorePatterns=\"src/integration\" \"scripts/bundleSize\" \"puppeteer\" \"ws-nextjs-app\"",
     "test:integration": "node src/integration/utils/runTests/index.js",
-    "test:integration:updatesnapshots": "node src/integration/utils/runTests/index.js  --updateSnapshot",
+    "test:integration:updatesnapshots": "yarn test:integration:express:updatesnapshots && yarn test:integration:nextjs:updatesnapshots",
```

Could we run these in parallel using run-p?

I've tried this locally & it seems OK, and is a tiny bit faster

```suggestion
    "test:integration:updatesnapshots": "run-p test:integration:express:updatesnapshots test:integration:nextjs:updatesnapshots",
```

---

Yea!